.github/workflows/checkstyle.yml

```diff
@@ -1,15 +1,11 @@
 name: Code Formatter
 
-on:
-  pull_request:
-  push:
-    branches:
-    - master
+on: [push]
 jobs:
   format:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1  # v1, NOT v2
+      - uses: actions/checkout@master
```

Comment:
1. The actions/checkout repo uses `main`, not `master` as its principal branch so that it can be politically correct.
2. The comment says that we want to use v1, not something later because we are pushing to a branch.
3. Sweeping error conditions under the rug with `|| true` is going to hide where problems arise in this process.